public/locales/en-US/send.ftl

```diff
@@ -85,5 +85,5 @@ footerLinkPrivacy = Privacy
 footerLinkTerms = Terms
 footerLinkCookies = Cookies
 requirePasswordCheckbox = Require a password to download this file
-addPasswordButton = Add Password
-incorrectPassword = Incorrect password. Try again?
+addPasswordButton = Add password
+incorrectPassword = Incorrect password. Try again.
```

Comment:
Unfortunately [this needs a new string ID](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings) if we want locales to update accordingly.

Reply:
thank you!